src/src/WebServer/WebServer.cpp

```diff
@@ -545,12 +546,20 @@ void getWebPageTemplateDefaultHeader(String& tmpl, const String& title, bool add
   {
     String tmp;
   #ifndef WEBPAGE_TEMPLATE_DEFAULT_HEADER
-    tmp = F("<header class='headermenu'><h1>ESP Easy Mega: {{title}}</h1><BR>");
+    tmp = F("<header class='headermenu'><h1>ESP Easy Mega: {{title}}"
+            #if BUILD_IN_WEBHEADER
+            "<div style='float:right;font-size:10pt'>Build: {{build}}</div>"
+            #endif // #if BUILD_IN_WEBHEADER
+            "</h1><BR>"
+            );
   #else // ifndef WEBPAGE_TEMPLATE_DEFAULT_HEADER
     tmp = F(WEBPAGE_TEMPLATE_DEFAULT_HEADER);
   #endif // ifndef WEBPAGE_TEMPLATE_DEFAULT_HEADER
 
     tmp.replace(F("{{title}}"), title);
+    #if BUILD_IN_WEBHEADER
+    tmp.replace(F("{{build}}"), parseString(get_binary_filename(), 4, '_')); // Assuming binary filename is ESP_Easy_mega_<date>_...
```

Comment:
I think this is tricky as it assumes the filename to be set.
But someone building with ArduinoIDE will not have this set.
Since you're relying on the date only, you can also output the build date, which is always set.